src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -117,6 +118,8 @@ public abstract class Index : ITombstoneAware, IDisposable, ILowMemoryHandler
 
         private readonly Size MappedSizeLimitOn32Bits = new Size(8, SizeUnit.Megabytes);
 
+        private const int IndexingCompressionMaxTestDocuments = 10000;
```

Comment:
Should use configuration, no?

Reply:
No, I looked at that, but it doesn't make any sense to increase that value after 10K.

Reply:
So why have the configuraiton option then?

Reply:
I moved it as a constant.